lstchain/scripts/lstchain_create_run_summary.py

```diff
@@ -161,23 +161,24 @@ def type_of_run(date_path, run_number, counters, n_events=500):
             pedestal_events = sum(
                 event.trigger.event_type == EventType.SKY_PEDESTAL for event in source
             )
-        with LSTEventSource(filename, config=config, max_events=n_events) as source:
-            source.log.setLevel(logging.ERROR)
             mono_events = sum(event.trigger.event_type == EventType.SUBARRAY for event in source)
```

Comment:
this is still pretty inefficient, since you still loop over all events twice. Please use something like:

```
from collections import Counter
event_type_counts = Counter(event.trigger.event_type for event in source)
```

Reply:
Thanks! Changed accordingly.